pkgs/applications/graphics/f3d/default.nix

```diff
@@ -30,5 +30,6 @@ stdenv.mkDerivation rec {
     license = licenses.bsd3;
     maintainers = with maintainers; [ bcdarwin ];
     platforms = with platforms; unix;
+    broken = true; # https://github.com/NixOS/nixpkgs/issues/262328
```

Comment:
Personally, I would not mind committing this as it is. [I find important to mark broken packages.]

But as this is nixpkgs and things are complicated. I suggest you to include error and current date of breakage.

For me, errors as:

> error while loading shared libraries:  libvtkInteractionWidgets.so.1: cannot open shared object file: No such file or directory

Please add information in previous line. And mention `Tracking issue: `.

Once you have updated, I can approve this.

Sorry about the extra work. I honestly would not mind information being incomplete. But some people do.

Reply:
> Sorry about the extra work.

We're talking 3 comments here, that's fine :wink: 

> I honestly would not mind information being incomplete, and it's debatable, I know. But some people do.

It makes sense, this information should be available in the referenced github issue, but the less we depend on github the better.

> Also, I'm not a commiter. So, I can only provide you my review. [Better than nothing I guess?]

Thanks for the review, it's probably easier to get attention on a reviewed PR :crossed_fingers: